smswebapp/test/test_admin.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test that the admin pages are available and protected.
+
+"""
+from urllib.parse import urlparse
+from django.contrib.auth.models import User
+from django.urls import reverse
+from django.test import TestCase
+
+
+class AdminTests(TestCase):
+    def setUp(self):
+        # create superuser
+        self.superuser = User.objects.create_superuser(
+            'test0001', 'test@example.com', '')
+
+    def test_unauthenticated(self):
+        """Unauthenticated log in to admin redirects to login."""
+        r = self.client.get(reverse('admin:index'))
+        self.assertEqual(r.status_code, 302)
```

Comment:
use assertRedirects?

Reply:
Can we do the urlparse trick with assertRedirects? I don't really want to have to hard-code the ``?next=`` behaviour.

Reply:
So, implemented in 4fac92e but it somehow feels like I'm hard-coding behaviour from the admin?